packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -847,6 +856,7 @@ class FinancialAssessment extends React.Component {
                                             onClick={() => this.toggleConfirmationModal(true)}
                                             is_disabled={
                                                 isSubmitting ||
+                                                has_completed_financial_assessment ||
```

Comment:
don't we want to disable it if FA is not completed?

Reply:
It currently only gets disabled once the form has been submitted by the user. I think the name may be causing some confusion. Should it be changed to "has_**submitted**_financial_assessment"?